plugins/tags.js

```diff
@@ -43,6 +43,12 @@ module.exports = function (corsica) {
     });
   });
 
+  corsica.on('tags.get', function(message) {
```

Comment:
```suggestion
  corsica.on('tags.getAll', function(message) {
```

Can we make this `tags.getAll`? I'd like to make it clear that this is getting the set, and this leaves room for per-tag info in the future.